pycbc/waveform/bank.py

```diff
@@ -307,6 +310,12 @@ def __init__(self, filename, approximant=None, parameters=None,
         self.extra_args = kwds
         self.ensure_hash()
 
+        # Variables for the injection-segment filter checker
```

Comment:
Need to think about this before responding, but I think it makes sense to expose a function for generating a "short" template.